### ISSUE TITLE
Doesn't show any error, warning or info

### ISSUE THREAD

**Issue by EliaBR** [AtomLinter/linter-phpcs#293]
Hey guys, sorry if I'm doing something wrong, but I couldn't make linter-phpcs show any error on atom. phpcs on terminal shows me what it should, but in atom it is just always 0 warning, 0 errors. Can you help me, please?

`phpcs --version` gives me
`PHP_CodeSniffer version 3.1.0 (stable) by Squiz (http://www.squiz.net)`
as well as `/home/user/.composer/vendor/bin/phpcs --version`.

_Linter: Debug_ on _Command Pallete_ shows the following:
```
Platform: linux
Atom Version: 1.20.1
Linter Version: 2.2.0
Opened file is ignored: No
Matching Linter Providers: 
  - PHPCS
Disabled Linter Providers: 
Standard Linter Providers: 
  - CSSLint
  - PHPCS
Indie Linter Providers: 
UI Providers: 
  - Linter
Ignore Glob: **/*.min.{js,css}
VCS Ignored Paths are excluded: true
Current File Scopes: 
  - *
  - text.html.php
  - meta.embedded.block.php
  - source.php
  - variable.other.php
```
Console on develop tools gives me nothing and busy-signal blinks as it should do.

My configs:
Exec path: `/home/elia/.composer/vendor/bin/phpcs`
`Search for exec` marked
Code standard: `PSR2` (tested with others without sucess)
`Disable when no config file` unmarked
`Search for configuration files` marked (I've tested with and wo conf files)
Tab width: `4`
`Show source` marked

I've installed CSSLint for testing purposes and it works.
